Contest_API.md

```diff
@@ -1557,6 +1561,7 @@ The request must fail with a 4xx HTTP status code if any of the following happen
 * A PATCH on an award id that doesn't exist.
 * A PATCH that contains an invalid attribute (e.g. null `citation` or `team_ids`).
 * A DELETE on an award id that doesn't exist.
+* A POST, PATCH, or DELETE on an award id that the server is managing.
```

Comment:
As alluded to above "is managing" is unclear.

Based on my suggestion above this should be:
> The request **may** fail [...] 
> * A POST, PATCH, or DELETE on an award id that the client is not managing

...or something to that effect.